packages/tao/src/shared/workspace.ts

```diff
@@ -157,8 +157,10 @@ export interface TargetConfiguration {
 export function workspaceConfigName(root: string) {
   if (existsSync(path.join(root, 'angular.json'))) {
     return 'angular.json';
-  } else {
+  } else if (existsSync(path.join(root, 'workspace.json'))) {
     return 'workspace.json';
+  } else {
+    return null;
   }
 }
```

Comment:
This was defaulting to 'workspace.json', it now returns null in workspace-less workspaces.